CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to contribute
+
+We welcome contributions from external contributors, and this document
+describes how to merge code changes into Psi4.  As of February 2016, the
+procedure for contributing code is exactly the same for the core development
+team and for external contributors.
+
+## Getting Started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* Fork the psi4/psi4 repository on GitHub.
+* Optionally, sign up for an account on [Travis CI](https://travis-ci.org/).
+
+## Making Changes
+
+* On your local machine, clone your fork of the psi4 repo, which is located at
+  github.com:yourusername/psi4
+* Add some really awesome code.  It's usually a good idea to make changes on
+  a new branch, but that is not mandatory.
+* If it's a new feature, you must add test cases and documentation.
+* When the code is ready to go, make sure you run the full test suite to make
+  sure nothing was broken.
+* If all tests pass, you can push the code to your fork of Psi4 on GitHub.  If
+  you signed up for a Travis CI account and linked it to your GitHub profile,
+  some rudimentary tests will automatically start.
+* Sign in to GitHub, navigate to your fork of Psi4 and submit a pull request.
+* If the tests all pass, on of the Psi4 admin team will merge the code into the
```

Comment:
You might also want to add the use of @psi4/p4pub-rw tag to alert everyone with psi4/psi4 PR access.